[2022-08-06 19:38:47,392] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-29T00:00:00+00:00 [queued]>
[2022-08-06 19:38:47,405] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-29T00:00:00+00:00 [queued]>
[2022-08-06 19:38:47,405] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:38:47,405] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 19:38:47,405] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:38:47,421] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-04-29 00:00:00+00:00
[2022-08-06 19:38:47,425] {standard_task_runner.py:52} INFO - Started process 2676 to run task
[2022-08-06 19:38:47,428] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-04-29T00:00:00+00:00', '--job-id', '824', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp0h0j_rnu', '--error-file', '/tmp/tmpq1828ol6']
[2022-08-06 19:38:47,428] {standard_task_runner.py:80} INFO - Job 824: Subtask second_task
[2022-08-06 19:38:47,492] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-29T00:00:00+00:00 [running]> on host b29c168c1666
[2022-08-06 19:38:47,566] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-29T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-29T00:00:00+00:00
[2022-08-06 19:38:47,566] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 19:38:47,567] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 19:38:47,577] {subprocess.py:85} INFO - Output:
[2022-08-06 19:38:47,578] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 19:38:47,579] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 19:38:47,603] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220429T000000, start_date=20220806T193847, end_date=20220806T193847
[2022-08-06 19:38:47,639] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 19:38:47,678] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:52:37,319] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-29T00:00:00+00:00 [queued]>
[2022-08-06 21:52:37,332] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-29T00:00:00+00:00 [queued]>
[2022-08-06 21:52:37,332] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:52:37,332] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:52:37,332] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:52:37,349] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-04-29 00:00:00+00:00
[2022-08-06 21:52:37,355] {standard_task_runner.py:52} INFO - Started process 2867 to run task
[2022-08-06 21:52:37,357] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-04-29T00:00:00+00:00', '--job-id', '824', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpik1qc77r', '--error-file', '/tmp/tmpu4htdcwv']
[2022-08-06 21:52:37,357] {standard_task_runner.py:80} INFO - Job 824: Subtask second_task
[2022-08-06 21:52:37,418] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-29T00:00:00+00:00 [running]> on host c3f731879890
[2022-08-06 21:52:37,501] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-29T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-29T00:00:00+00:00
[2022-08-06 21:52:37,502] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:52:37,502] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:52:37,514] {subprocess.py:85} INFO - Output:
[2022-08-06 21:52:37,515] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:52:37,515] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:52:37,545] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220429T000000, start_date=20220806T215237, end_date=20220806T215237
[2022-08-06 21:52:37,568] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:52:37,595] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:58:06,186] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-29T00:00:00+00:00 [queued]>
[2022-08-06 21:58:06,197] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-29T00:00:00+00:00 [queued]>
[2022-08-06 21:58:06,197] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:58:06,197] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:58:06,197] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:58:06,214] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-04-29 00:00:00+00:00
[2022-08-06 21:58:06,222] {standard_task_runner.py:52} INFO - Started process 2672 to run task
[2022-08-06 21:58:06,225] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-04-29T00:00:00+00:00', '--job-id', '826', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp6fts3kc2', '--error-file', '/tmp/tmpwedxvf0_']
[2022-08-06 21:58:06,225] {standard_task_runner.py:80} INFO - Job 826: Subtask second_task
[2022-08-06 21:58:06,310] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-29T00:00:00+00:00 [running]> on host bc7e154a4fdd
[2022-08-06 21:58:06,418] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-29T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-29T00:00:00+00:00
[2022-08-06 21:58:06,419] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:58:06,419] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:58:06,435] {subprocess.py:85} INFO - Output:
[2022-08-06 21:58:06,437] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:58:06,437] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:58:06,476] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220429T000000, start_date=20220806T215806, end_date=20220806T215806
[2022-08-06 21:58:06,516] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:58:06,561] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:13:06,036] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-29T00:00:00+00:00 [queued]>
[2022-08-06 22:13:06,050] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-29T00:00:00+00:00 [queued]>
[2022-08-06 22:13:06,050] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:13:06,050] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:13:06,050] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:13:06,073] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-04-29 00:00:00+00:00
[2022-08-06 22:13:06,080] {standard_task_runner.py:52} INFO - Started process 2678 to run task
[2022-08-06 22:13:06,083] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-04-29T00:00:00+00:00', '--job-id', '825', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp8orfhs7w', '--error-file', '/tmp/tmp2kqscc_g']
[2022-08-06 22:13:06,083] {standard_task_runner.py:80} INFO - Job 825: Subtask second_task
[2022-08-06 22:13:06,155] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-29T00:00:00+00:00 [running]> on host 1250c3b659cd
[2022-08-06 22:13:06,246] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-29T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-29T00:00:00+00:00
[2022-08-06 22:13:06,247] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:13:06,247] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:13:06,262] {subprocess.py:85} INFO - Output:
[2022-08-06 22:13:06,263] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:13:06,264] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:13:06,297] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220429T000000, start_date=20220806T221306, end_date=20220806T221306
[2022-08-06 22:13:06,334] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:13:06,370] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:24:24,821] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-29T00:00:00+00:00 [queued]>
[2022-08-06 22:24:24,832] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-29T00:00:00+00:00 [queued]>
[2022-08-06 22:24:24,832] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:24:24,832] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:24:24,832] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:24:24,846] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-04-29 00:00:00+00:00
[2022-08-06 22:24:24,852] {standard_task_runner.py:52} INFO - Started process 2725 to run task
[2022-08-06 22:24:24,855] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-04-29T00:00:00+00:00', '--job-id', '826', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp1m5d6rwe', '--error-file', '/tmp/tmpftum0e11']
[2022-08-06 22:24:24,855] {standard_task_runner.py:80} INFO - Job 826: Subtask second_task
[2022-08-06 22:24:24,957] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-04-29T00:00:00+00:00 [running]> on host b215076695c3
[2022-08-06 22:24:25,035] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-04-29T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-29T00:00:00+00:00
[2022-08-06 22:24:25,036] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:24:25,036] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:24:25,049] {subprocess.py:85} INFO - Output:
[2022-08-06 22:24:25,050] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:24:25,051] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:24:25,084] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220429T000000, start_date=20220806T222424, end_date=20220806T222425
[2022-08-06 22:24:25,106] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:24:25,140] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
